rfc021-platform-policy.md

```diff
@@ -6,32 +6,30 @@ Type: Informational
 ---
 
 
-# Chef Platform Support Policy
+# Chef Platform Policy
 
 The purpose of this RFC is to clarify:
 
-* What specific operating system platforms and platform versions are supported by the software produced by Chef Software, Inc.
-* What is the meaning of "supported platform"
-
-This RFC does *not* address "What is the product lifecycle of Chef Software, Inc.'s software". That is covered in a separate RFC.
+* What specific operating system platforms and platform versions will work with the software produced by Chef Software, Inc.
+* What are the meanings of "foundational platform", "secondary platform", and "other platforms"
 
 ## Chef Client
 
-A Chef Client supported platform means:
+To be included as either a Foundational Platform or a Secondary Platform, the following must be true for that specific platform and version:
 
 * Omnitruck won't fail when confronted with the platform and version
 * The most important core resources (package, service, template) work out of the box
 * Ohai attributes for ```platform```, ```platform_family```, ```platform_version``` and ```kernel.machine``` are correct
 
-Chef Client support policies also apply to Ohai, since that is a dependency.
+These policies also apply to Ohai, since that is a dependency.
 
-### Tier 1 Support
+### Foundational Platforms
 
-Tier 1 supported platforms are those for which Chef builds native binary "Omnitruck" (full-stack installer) packages. For each platform, Chef performs some post-build verification on them or their equivalents. For example, we may elect to do post-build verification for Oracle Enterprise Linux using the same test results as Red Hat Enterprise Linux, since they are so similar). A platform must have a Lieutenant before being added to this tier.
+Foundational platforms are those for which native binary "Omnibus" (full-stack installer) packages are built and available via Omnitruck for every release. For each platform, Chef performs some post-build verification on them or their equivalents. For example, we may elect to do post-build verification for Oracle Enterprise Linux using the same test results as Red Hat Enterprise Linux, since they are so similar. Only platform versions that receive automated post-build verification are listed here, even when newer or intermediate versions may be known to work. A platform must have a Lieutenant as defined in RFC 030 before being added to this tier.
 
 Platform | Versions | Architectures | Package Format | Built on
 --- | --- | --- | --- | ---
-AIX | 6.1, 7.1, 7.2 | ppc64 | bff | AIX 6.1
+AIX | 6.1, 7.1 | ppc64 | bff | AIX 6.1
```

Comment:
Nothing special, it just works. It's different enough from 7.1 that it's worth mentioning (IBM has a big thing about "forward binary compatibility" on AIX).